study.md

```diff
@@ -40,14 +40,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+to tell that we want to modify something
```

Comment:
Specifically, to modify the object that the method was invoked upon rather than return a copy.